layouts/filter.tsx

```diff
@@ -1,15 +1,25 @@
-import React from "react";
+import React, { useEffect } from "react";
+import { useRouter } from "next/router";
 
 import Footer from "components/organisms/Footer/footer";
 import Header from "components/organisms/Header/header";
 import Nav from "components/organisms/ToolList/nav";
 import TopNav from "components/organisms/TopNav/top-nav";
 import FilterHeader from "components/molecules/FilterHeader/filter-header";
 
+import { setQueryParams } from "lib/utils/query-params";
 import useNav from "lib/hooks/useNav";
 
 const FilterLayout = ({ children }: { children: React.ReactNode }) => {
   const { toolList, selectedTool, filterName, selectedFilter, userOrg } = useNav();
+  const router = useRouter();
+  const { range } = router.query;
+
+  useEffect(() => {
+    if (!range) {
+      setQueryParams({ range: "30" });
+    }
+  }, [range, router.isReady]);
```

Comment:
Refactor (potentially blocking): This seems unnecessary as the default value of 30 would be used if none was present in the querystring.

```suggestion
```